gridcapa-core-cc-post-processing-app/src/main/java/com/farao_community/farao/core_cc_post_processing/app/CoreCCPostProcessingHandler.java

```diff
@@ -53,20 +53,18 @@ public Consumer<Flux<TaskDto>> consumeTaskDtoUpdate() {
      */
     private void postProcessFinishedTasks(TaskDto taskDtoUpdated) {
         try {
+            LOGGER.debug("Task {} updated.", taskDtoUpdated);
```

Comment:
`TaskDto` doesn't have an overriden `toString()` method. In order to have intelligible information in your logs, you should display its `id` or `timestamp` value rather than the taskDto itself.